jest.config.js

```diff
@@ -23,6 +23,7 @@ const config = {
   ],
   testPathIgnorePatterns: [
     "<rootDir>/frontend/.*/.*.tz.unit.spec.{js,jsx,ts,tsx}",
+    "<rootDir>/release/.*",
```

Comment:
It looks like another jest config will better separate relations between packages inside repo

Reply:
I'm not sure I understand. We actually do have a separate jest config in the release directory, but without excluding that directory from the parent config, it's gonna look for tests in the release directory too.

Reply:
oh right my bad